app/views/players/index.html.slim

```diff
@@ -6,9 +6,15 @@
 
   .alert.alert-secondary.mt-4
     h5.mb-3 Register New Player
-    = simple_form_for :player, url: tournament_players_path(@tournament), html: { class: 'form-inline d-block' } do |f|
+    = simple_form_for :player, url: tournament_players_path(@tournament), html: { class: 'form-inline d-block register-player' } do |f|
       = render 'form', f: f, organiser_view: true, new_registration: true
       .mt-2.text-right
+        a.btn.btn-link.text-info.mr-2.assign-table-number onclick="assignTableNumber()"
+          => fa_icon 'wheelchair'
```

Comment:
I'm not sure we want to use this icon.  How about  sticky-note or bookmark? If OP or EDI is ok with this, then I am as well, though.

Reply:
I was wondering about that. I couldn't find another icon that made sense, but I'll switch it.

Reply:
Done